bin/init-adcirc.sh

```diff
@@ -409,12 +409,17 @@ function dumpJSON()
     local MODULE_LIST=$(module list 2>&1 | grep '1)');
 
     # get compiler info
-    local _FC=$(which ifort)
-    local _CC=$(which icc)
+    local _FC=$(which ifort 2> /dev/null)
```

Comment:
May want to make this `ifx`

Reply:
Most platforms are still `ifort` so this looks reasonable to me.